packages/standalone/capabilities/bundle/config/upstream/capabilities-crd.yaml

```diff
@@ -0,0 +1,326 @@
+---
```

Comment:
non-blocking: I wonder if we should symlink this to the file in `config/crd/bases`? Is that a good practice? Duplicate files mean it's easy to miss updating this file.